### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#254] [Feedback]
![JG RTX block textures](https://github.com/user-attachments/assets/eed6c30d-d28c-4874-8dc8-a0749a06def3)

## Changelog
**Added**
- Default fog is back!

**Updated**
- Glowstone
- Prismarine dark (Bedrock)
- Prismarine bricks
- Emerald block
- Blackstone
- Lapis block
- Stone brick (Bedrock)
- Brick (Bedrock)
- Acacia planks (Bedrock)

**Fixed**
- Red sandstone side texture

## What's Changed
* patch: Normalize normal maps by @jasonjgardner in https://github.com/jasonjgardner/jg-rtx/pull/250
* upd: Retry Java conversion by @jasonjgardner in https://github.com/jasonjgardner/jg-rtx/pull/251
* fix(bedrock): Reconstruct Z in normal maps by @jasonjgardner in https://github.com/jasonjgardner/jg-rtx/pull/252
* fix(bedrock): Redo reconstruct Z by @jasonjgardner in https://github.com/jasonjgardner/jg-rtx/pull/253

**Full Changelog**: https://github.com/jasonjgardner/jg-rtx/compare/v1.0.0...v1.1.0

---
### New Add-on Content
![JG RTX extra bricks](https://github.com/user-attachments/assets/9dcd5895-0e16-4863-8e5e-b13977c2cd20)
17 decorative stone brick block variations: https://jasonjgardner.gumroad.com/l/jgrtx-extra-bricks

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by JackTheFragger** [comment]
I've tried several RTX ressource packs. PrizmaRTX has a lightning torch in hand so dark environments can be lightened without placing torches but only carry it in your hand. It would be nice to have it in your ressource pack, which is the best looking one!

P.S. I didn't find a feature request option here.
